packages/ketcher-react/src/script/editor/Editor.ts

```diff
@@ -573,9 +573,9 @@ class Editor implements KetcherEditor {
     this.historyStack[this.historyPtr] = action;
 
     if (this._tool instanceof toolsMap.paste) {
-      this.event.change.dispatch();
+      ketcherProvider.getKetcher().changeEvent.dispatch();
     } else {
-      this.event.change.dispatch(action);
+      ketcherProvider.getKetcher().changeEvent.dispatch(action);
```

Comment:
For a bit of readability I'd prefer to have some variable assigned to ketcher instance and it being reused as long as function uses ketcher more than once